confidant/cli/secrets.py

```
@@ -0,0 +1,101 @@
+
+# confidant/cli/secrets.py
+
+import typer
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)